### DISCUSSION TITLE
Немного вопросов.

### DISCUSSION THREAD

**Discussion by Murakumo-JP** [xtrime-ru/antizapret-vpn-docker#34] [Q&A] [answered]
Здраствуйте снова, понимаю что возможно заколебал вас уже но так как я глупый в этом вопросе нечего не могу поделать, хочу задать всего 4 вопроса.
1. в консоли сервера вижу вот такое это нормально или нет ?
![image](https://github.com/user-attachments/assets/8dbf778f-0ea7-4f24-b303-58b6eafe03c2)
2. Можно ли как то создать дополнительные файлы конфигураций для клиентов ?
3. Можно ли как то установить дополнительно OpenVPN Management interface чтоб как то это всё через панель контролировать или в случай с антизапретом это не работает ?
4. Есть какой то фикс для мобильного интернета у меня проста не подключается к серверу вечно разрыв идёт и передисконект идёт.

**Comment by xtrime-ru** [comment]
Я, к сожалению, не имею достаточно опыта по работе с openvpn, поэтому не могут ответить на эти вопросы. 
Внутри контейнера используется ubutnu 24.04 и openvpn 2.6.9. Любые инструкции из сети должны помочь.

Так же можно уточнить на форуме https://ntc.party/

**Comment by xtrime-ru** [comment]
> 4\. Есть какой то фикс для мобильного интернета у меня проста не подключается к серверу вечно разрыв идёт и передисконект идёт.

В новой версии добавили патч от Tunnelblick для дополнительной обфускации. Просьба поставить ENV переменную SCRAMBLE=1 в docker-compose.yml, перезапустить контейнер и скачать новый ovpn.

**Reply by Murakumo-JP** [reply]
> > 4\. Есть какой то фикс для мобильного интернета у меня проста не подключается к серверу вечно разрыв идёт и передисконект идёт.
> 
> В новой версии добавили патч от Tunnelblick для дополнительной обфускации. Просьба поставить ENV переменную SCRAMBLE=1 в docker-compose.yml, перезапустить контейнер и скачать новый ovpn.

Круто, спасибо, наверное с нуля всё подниму со свежей версий.

**Reply by xtrime-ru** [reply, marked as answer]
Возможно еще TLS_CRYPT=1 поможет. Надо тоже проверить.

**Reply by xtrime-ru** [reply]
Добавил wireguard сервер. Попробуйте его использовать. Он может быть стабильнее, чем ovpn
Используется wg-easy, там и отдельные ключи можно генерировать и время жизни устанавливать и статисктику по клиентам смотреть. Все через панель управления.

**Reply by Murakumo-JP** [reply]
Попытался я wireguard поставить на сервере вроде всё работает но не через приложения на телефоне не с Кинетика до сервера нет соединения, так не смог решить эту проблему. 😥 откатился обратно до OpenVPN блага на роутере он работает пока.

**Reply by xtrime-ru** [reply]
@Murakumo-JP Можно одновременно использовать несколько VPN протоколов. Возможно ваш провайдер блочит wireguard. Просьба попробовать новую версию с контейнером amnezia-wireguard, оно лучше сопротивляется блокировкам и при этом обычные клиенты могут использовать этот сервер.

**Reply by Murakumo-JP** [reply]
> @Murakumo-JP Можно одновременно использовать несколько VPN протоколов. Возможно ваш провайдер блочит wireguard. Просьба попробовать новую версию с контейнером amnezia-wireguard, оно лучше сопротивляется блокировкам и при этом обычные клиенты могут использовать этот сервер. 
> 

Хорошо попробую завтра. 👍
